src/SimpleExcelReader.php

```diff
@@ -314,15 +325,17 @@ protected function getSheet(): SheetInterface
         $this->reader->open($this->path);
 
         foreach ($this->reader->getSheetIterator() as $key => $sheet) {
```

Comment:
I think what you're doing here is make sure the requested sheet name is that active one. If so extract this to a function with that name.